xbmc/GUIInfoManager.cpp

```diff
@@ -6274,22 +6274,22 @@ std::string CGUIInfoManager::GetLabel(int info, int contextWindow, std::string *
     }
     break;
   case VIDEOPLAYER_AUDIO_LANG:
-    if(g_application.m_pPlayer->IsPlaying())
+    if(g_application.GetAppPlayer()->IsPlaying())
     {
       strLabel = m_audioInfo.language;
     }
     break;
   case VIDEOPLAYER_STEREOSCOPIC_MODE:
-    if(g_application.m_pPlayer->IsPlaying())
+    if(g_application.GetAppPlayer()->IsPlaying())
     {
       strLabel = m_videoInfo.stereoMode;
     }
     break;
   case VIDEOPLAYER_SUBTITLES_LANG:
-    if(g_application.m_pPlayer && g_application.m_pPlayer->IsPlaying() && g_application.m_pPlayer->GetSubtitleVisible())
+    if(g_application.GetAppPlayer() && g_application.GetAppPlayer()->IsPlaying() && g_application.GetAppPlayer()->GetSubtitleVisible())
```

Comment:
here it is even worse. you could get a ref to player and then use this for the following calls.